File: rectangulo.cpp
#include <iostream>

class Rectangulo {
private:
    double longitud;
    double ancho;

public:
    
    Rectangulo(double l, double a) : longitud(l), ancho(a) {}

    
    double calcular_area() {
        return longitud * ancho;
    }

  
    double calcular_perimetro() {
        return 2 * (longitud + ancho);
    }

   
    void cambiar_longitud(double nueva_longitud) {
        longitud = nueva_longitud;
    }

   
    void cambiar_ancho(double nuevo_ancho) {
        ancho = nuevo_ancho;
    }
};

int main() {
    
    Rectangulo Rectangulo_inicial(9, 6);

 
    std::cout << "el area del rectángulo es: " << Rectangulo_inicial.calcular_area() << std::endl;

    std::cout << "el perímetro del rectángulo es: " << Rectangulo_inicial.calcular_perimetro() << std::endl;

    return 0;
}

File: rectangulo.py
class Rectangulo:
    def __init__(self, longitud, ancho):
        self.longitud = longitud
        self.ancho = ancho

    def calcular_area(self):
        return self.longitud * self.ancho

    def calcular_perimetro(self):
        return 2 * (self.longitud + self.ancho)

    def cambiar_longitud(self, nueva_longitud):
        self.longitud = nueva_longitud

    def cambiar_ancho(self, nuevo_ancho):
        self.ancho = nuevo_ancho


rectangulo = Rectangulo(8, 4)


print("el area del perimetro es:", rectangulo.calcular_area())


print("el perimetro del rectangulo es:", rectangulo.calcular_perimetro())
